gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-notifications.spec.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ADMIN_USER, API_PUBLISHER_USER } from '@fakers/users/users';
+import { ApisFaker } from '@gravitee/fixtures/management/ApisFaker';
+import { ApiImport } from '@model/api-imports';
+import apiDetails from 'ui-test/support/PageObjects/Apis/ApiDetails';
+import { PlanStatus } from '../../../../../lib/management-webclient-sdk/src/lib/models';
+import { PlansFaker } from '@gravitee/fixtures/management/PlansFaker';
+
+let api: ApiImport;
+
+describe('Notifications page', () => {
+  beforeEach(() => {
+    cy.loginInAPIM(ADMIN_USER.username, ADMIN_USER.password);
+    cy.visit(`/#!/environments/default/apis/${api.id}/notification-settings`);
+    cy.contains('Portal Notifications').should('be.visible');
+  });
+
+  before(() => {
+    cy.log('Import (create) v2 API');
+    cy.request({
+      method: 'POST',
+      url: `${Cypress.env('managementApi')}${Cypress.env('defaultOrgEnv')}/apis/import`,
+      auth: { username: API_PUBLISHER_USER.username, password: API_PUBLISHER_USER.password },
+      body: ApisFaker.apiImport({
+        plans: [PlansFaker.plan({ status: PlanStatus.PUBLISHED })],
+      }),
+    }).then((response) => {
+      expect(response.status).to.eq(200);
+      api = response.body;
+    });
+  });
+
+  const ApiDetails = new apiDetails();
+
+  it('Verify notification page elements', () => {
+    cy.getByDataTestId('api_notifications_add_notification').should('be.visible');
+    cy.contains('Portal Notifications').should('be.visible');
+    cy.contains('Default Mail Notifications').should('be.visible');
+  });
+
+  it('Create generic notification', () => {
+    cy.contains('Default Mail Notifications').should('be.visible');
+    cy.getByDataTestId('api_notifications_add_notification').click();
+    cy.contains('New notification', { timeout: 5000 }).should('be.visible');
+    cy.getByDataTestId('api_notifications_create_namefield').type('Test notification');
+    cy.getByDataTestId('create_notification_button').contains('Create').click();
+    cy.contains('Notification created successfully').should('be.visible');
+    cy.contains('Test notification').should('be.visible');
+  });
+
+  it('Delete generic notification', () => {
+    cy.getByDataTestId('api_notifications_delete_notification').first().click();
```

Comment:
I noticed a lot of `.first()` in this test set. Try to avoid it as much as possible as it makes the test more fragile.
![Screenshot 2023-11-24 at 16 35 01](https://github.com/gravitee-io/gravitee-api-management/assets/15231507/8fa54451-abcf-4c1c-b821-56cfcf371fa3)
With `.first()` this test breaks if _Portal Notifications_ becomes deletable in the future or the item order changes.